packages/libs/client/etc/client.api.md

```diff
@@ -21,9 +21,26 @@ export function buildCommandFromTemplate(parts: string[], holes: string[], args:
 // @internal (undocumented)
 export function buildUnsignedTransaction(parts: string[], holes: string[], args: Record<string, string>): IPactCommand & ICommandBuilder<{}>;
 
+// @alpha
+export class ContCommand extends PactCommand implements IContCommand, IContCommandBuilder<Record<string, unknown>> {
+    constructor(proof: string, step: number, pactId: string, rollback: boolean);

```

Comment:
Javad
_I think its better to pass (proof, step, pactId, rollback) to the constructor, then you don't need those related setters.
and the API would be more cleaner because we want to set those properties in one step anyway_

Reply:
Changed made